assignments/callbacks.js

```diff
@@ -4,12 +4,19 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 /* 
 
+*/
   //Given this problem: 
   
   function firstItem(arr, cb) {
-    // firstItem passes the first item of the given array to the callback function.
+// firstItem passes the first item of the given array to the callback function.
+    return cb(arr[3])
+    
+    
+
   }
 
+firstItem(items, console.log)
+
```

Comment:
You would need to write out the function here instead. Rather than type `console.log` here you would need to replace it with: `(item1) => {console.log(item1)}`